### DISCUSSION TITLE
Help with and SMTP to send notifications.

### DISCUSSION THREAD

**Discussion by kelabraham** [PHPMailer/PHPMailer#2944] [Q&A]
Working on this project and I'm stuck guys, I need a SMTP that can help send notifications as soon as someone submits a request. And when the notification is sent it should contain a link that sends the authorizer to a page where he has to approve or ignore the request.
I have tried a couple of SMTP's like Brevo and Phpmailer, but i can't find my way around the Phpmailer, but the Brevo API code has some errors.

**Comment by Synchro** [comment]
PHPMailer only concerns itself with sending email. The rest is up to you. The fact that it's a request that needs approval and has pages to manage it is not something that PHPMailer can do anything about. You'll need to write handlers that set up database records representing the request, and then write the code for the operations that change its status. That kind of thing is usually done with a Framework, which can make things much easier to build; take a look at Laravel.
